pkg/generator/helper/helper.go

```diff
@@ -180,43 +180,138 @@ func Test{{.CobraCmdFuncName}}(t *testing.T) {
 
 	testCmd := {{.CobraCmdFuncName}}()
 
-	// only high level testing performed - details are tested in step generation procudure
+	// only high level testing performed - details are tested in step generation procedure
```

Comment:
Review would be easier with less files changed due to this typo fix.

Reply:
Ok, will revert and put into a separate PR.